### PR TITLE
fix(config): replace void function +yas-active-p

### PR DESCRIPTION
**Pull request by LemonBreezes**
Just noticed a function I had removed from the code for my previous TAB PR had somehow gotten in.

-----
- [x] I searched the issue tracker and this hasn't been PRed before.
- [x] My changes are not on [the do-not-PR list](https://doomemacs.org/d/do-not-pr) for this project.
- [x] My commits conform to [the git conventions](https://doomemacs.org/d/git-conventions).
- [x] My changes are visual; I've included before and after screenshots.
- [ ] I am blindly checking these off.
- [x] Any relevant issues or PRs have been linked to.
- [ ] This a draft PR; I need more time to finish it.

### CODE REVIEW

File: modules/config/default/+evil-bindings.el [hunk 1]
```diff
@@ -46,7 +46,8 @@
                 (doom-lookup-key (kbd "TAB") overriding-terminal-local-map))
             cmd)
            ,@(when (modulep! :editor snippets)
-               '(((+yas-active-p)
+               '(((memq (bound-and-true-p yas--active-field-overlay)
+                   (overlays-in (1- (point)) (1+ (point))))
```
Comment (hlissner) [inline modules/config/default/+evil-bindings.el:50]: Please correct the indentation. `(overlays-in ...` should align with `(bound-and-true-p ...)`.

**Review by hlissner — CHANGES_REQUESTED** [review]

**Review by hlissner — CHANGES_REQUESTED** [review]
Also, please correct the scope in the commit message. I.e. should be `fix(default): ...`.

File: modules/config/default/+evil-bindings.el (hunk 1 above)
Comment (LemonBreezes) [reply modules/config/default/+evil-bindings.el:50]: Ok. Just fixed it.

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Thanks again for your help!

**Pull request merged**